### ISSUE TITLE
TinyXML++ experience poor, will now try CodeSynthesis XSD

### ISSUE THREAD

**Issue by GoogleCodeExporter** [wxFormBuilder/ticpp#46]
```

I thought that I would write here (since there is no forum) and mention
that my experience with TinyXML++ was very poor.  I was looking for a quick
and easy solution for reading/writing C++ object configuration files.  As
such I came across TinyXML++ and CodeSynthesis XSD.  The latter being a XML
Data binder for C++.  I read on both and decided upon TinyXML++ simply
because I thought CodeSynthesis required more configuration, making my
project as a whole less portable and more hassle to work with at future
dates if there are interruptions...i.e. I hate working with something and
then coming back to it months later and having to figure out again how to
run a separate compiler.   I strive for simplicity in configuration, and
use.   So I tried TinyXML++ and now have decided to abandon it.  Here is my
brief assesment of TinyXML++:

Documentation is very poor.  Premake instructions are not up to date, and
examples in the tutorial are incomplete.
Source commenting is almost non existent.
Was able to build lib file with MSVS but then got exception, in my program,
when trying to load a test XML file.
Tried to build TinyXML as part of my program but ran into numerous errors.

To sum up, way too much difficulty involved to mess further with,
especially for something that is supposed to be minimalistic.   As a
programmer I forgive a lack of good documentation etc. only if they just
work and have zero defects.



```

Original issue reported on code.google.com by `bhar...@gmail.com` on 23 Apr 2009 at 10:13
